CIP-dapp-bridge-rework/README.md

```diff
@@ -0,0 +1,1169 @@
+
+8. **Security**: Currently, DApps are given too much responsibility in the formation of transactions. This could potentially become an attack vector on a user who is confident in their security. Besides the risk of leaking wallet data, including addresses, there is a possibility that a DApp could form a malware transaction. This transaction might include fields not checked by the wallet but still obtain all necessary signatures. For example, the user could unknowingly become part of a criminal scheme or lose funds.<br><br>
+
+9. **Trustful system**: While blockchain is inherently a trustless system, it seems very odd when a wallet unconditionally trusts all user data to a third party at the first request.<br><br>
+
+By solving these problems, we will lower the entry barrier for DApp developers into the ecosystem, while simultaneously enhancing the security of the end-user who has entrusted their funds to a wallet, but should not trust their management to just any application they encounter. The wallet will act as a black box, and the DApp as an extension of its UI, not a replacement of its engine.
+
+Additionally, this approach will simplify the integration of wallets and DApps from the wallet developers' perspective, as it will provide a more natural interface for interaction, easily translatable into native scenarios of interaction with the ordinary user through the wallet's own UI.
+
+## Specification
+
+### The main idea
+The main idea is to delegate as many operations as possible to the wallet, which are its core functions. This will greatly simplify the lives of DApps developers. It will also normalize the transaction pipeline, which is currently highly intermingled between the DApp and the wallet.
+We are moving away from imperative interaction with the wallet towards a declarative approach, where the DApp does not have to assemble the transaction itself; it should be sufficient to **describe** the desired outcome. Just as a regular wallet user is not required to delve into all the details - they simply specify the address, amount, and press a button. This is literally why software wallets exist.
+For example, here is a description of a transaction sending 1 ada to some address in the testnet:
+
+```
+{
+    outputs:[{
+        address: "addr_test1qpy486p7jhq933kay7rf78jz2j6k5ya7u95nnnjetgj9r02dp276p7j8023vmum9wu8gp7q54f3rjke45j0klk3pmwvsgyx4n7",
+        value: {
+            lovelaces: 1000000
+        }
+    }]
+}
+```
+
+Or a description of a delegation transaction to a certain pool:
+
+
+```
+{
+    certificates: [{
+        type: "stakeDelegation",
+        stakeKeyId: 0,
+        poolHash: "660f2b816e90e4e826daebf33357d76ab89414f88d57dd7d9f7898fe"
+    }]
+}
+```
+
+
```

Comment:
```suggestion

```
Parts of the document have a loose spacing which doesn't work well between the number of headings & length of the document.  (Some of the sections with shorter content might even work better as bulleted lists or tables.)

This is completely optional & bottom priority for review, especially considering the weighty matters of scope & purpose currently under debate.

Reply:
The document is not final and could be improved several times based on community feedback so I think, we can delay it until the finalization phase.

Reply:
resolving as per https://github.com/cardano-foundation/CIPs/pull/623#discussion_r1408201380 in favour of a denser markup format (e.g. lists or tables), so we'll keep that one open instead.